packages/system/src/components/homepage/contributor-banner.css.ts

```diff
@@ -29,7 +29,7 @@ export const contributorImageStyle = style([
 		borderStyle: "solid",
 		selectors: {
 			"& + &": {
-				marginLeft: pxToRem(-14),
+				marginLeft: pxToRem(5),
 			},
 		},
```

Comment:
You should remove this all together because it is creating a vertical alignment issue for the top row of images and the rest of the rows. 

Right now it looks like this
<img width="195" alt="Screen Shot 2022-12-12 at 2 22 39 PM" src="https://user-images.githubusercontent.com/67210629/207167887-2b907c59-02aa-4147-b81c-a6fc8a6aaec1.png">


Once you remove that section then, it will align nicely
<img width="162" alt="Screen Shot 2022-12-12 at 2 23 29 PM" src="https://user-images.githubusercontent.com/67210629/207168004-ff3e9636-687b-4d29-ab34-1a773c22e1f9.png">